src/HEXONET/Response.php

```diff
@@ -298,13 +299,11 @@ public function getCommand()
      */
     public function getCommandPlain()
     {
-        $tmp = "";
-        foreach ($this->command as $key => $val) {
-            $tmp .= "$key = $val\n";
-        }
-        return $tmp;
+        CommandFormatter::getSortedCommand($this->command);
```

Comment:
Is that line even of use? It returns of course the updated sorted version, but you don't consider dealing with that at all. I am wondering!